Application/Frontend/pages/accord.tsx

```diff
@@ -186,7 +188,8 @@ export default function Accord() {
               .map((_, index) => (
                 <Skeleton key={index} h={30} mt="sm" animate={false} />
               ))} */}
-              <ServerList/>
+              {/* <ServerList/> */}
+              <MemberList/>
```

Comment:
We are going to implement this into our in-server.tsx. But for demonstration purposes, this is the product of itself. You'd need to change it back to displaying ServerList, since we are navigating from this. I would assume that you are doing this just for demonstration purposes. Make sure to change the components back to ServerList, then we can do an approval.